### PR TITLE
refactor: simplify `sortScripts` function

### PR DESCRIPTION
**Pull request by fisker**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,20 +28,38 @@ const defaultNpmScripts = [
...
+
+  const sortOrder = scriptCompareKeys
+    .reduce(
+      (sortOrder, key) =>
+        sortOrder.concat(
+          prefixableScripts.has(key) ? [`pre${key}`, key, `post${key}`] : [key],
+        ),
+      [],
+    )
+    // TODO: Remove next line when/if this PR gets merged: https://github.com/keithamus/sort-object-keys/pull/11
```
Comment (keithamus) [inline index.js]: This was merged and released as `v1.1.3`!

File: index.js [hunk 2]
```diff
@@ -28,20 +28,38 @@ const defaultNpmScripts = [
...
+      }
+      return script
+    })
+    .sort()
+
+  const sortOrder = scriptCompareKeys
+    .reduce(
+      (sortOrder, key) =>
+        sortOrder.concat(
+          prefixableScripts.has(key) ? [`pre${key}`, key, `post${key}`] : [key],
```
Comment (keithamus) [inline index.js]: Clever. I like it.

**Comment by keithamus** [comment]
Let's update now `v1.1.3` of `sort-object-keys` is out and I'll merge!

**Comment by fisker** [comment]
done.

**Pull request merged**